cirq-core/cirq/experiments/fidelity_estimation.py

```diff
@@ -141,7 +141,7 @@ def xeb_fidelity(
     circuit: cirq.Circuit,
     bitstrings: Sequence[int],
     qubit_order: QubitOrderOrList = QubitOrder.DEFAULT,
-    amplitudes: Mapping[int, complex] | None = None,
+    amplitudes: np.ndarray | None = None,
```

Comment:
@pavoljuhas This change is not backwards compatible, as not all `Mapping` types are `np.ndarray`s.

As a specific example, `dict[int, complex]` worked with the previous signature, but fails now.

Reply:
@95-martin-orion - yes, that is not right, thank you for flagging this!

This should be fixed by #7584.